subprojects/docs/src/docs/release/notes.md

```diff
@@ -100,7 +101,27 @@ gradle mAL:cT
 
 Note, that even though the kebab case name matching works with tasks too, the recommendation is still to use camel case for them. 
 
-To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).  
+To learn more about name abbreviation, check out the [user guide](userguide/command_line_interface.html#task_name_abbreviation).
+
+## Support for version ranges in repository content filtering
+
+With [repository content filtering](userguide/declaring_repositories.html#sec:repository-content-filtering), builds can control which repositories are queried for which dependency.
+This feature provides performance and security benefits.
+
+With this release, when including or excluding a specific dependency version, the build author can use a version range:
```

Comment:
Is there any use case that we can mention here? I understand what the feature does, I just have no idea why I would do it.

Reply:
This is mostly going to be used for internal dependencies or allowing / preventing use of some versions following company policies.
But I do not think this is worth describing here.

Reply:
Ok with not attempting to add anything there?